homeassistant/components/netatmo.py

```diff
@@ -6,46 +6,49 @@
 """
 import logging
 from urllib.error import HTTPError
+
+import voluptuous as vol
+
 from homeassistant.const import (
     CONF_API_KEY, CONF_PASSWORD, CONF_USERNAME)
-from homeassistant.helpers import validate_config, discovery
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
 
 REQUIREMENTS = [
     'https://github.com/jabesq/netatmo-api-python/archive/'
     'master.zip#lnetatmo==0.5.0']
+# 'v0.5.0.zip#lnetatmo==0.5.0']
```

Comment:
I hoped that the archives are back.

https://github.com/philippelt/netatmo-api-python/issues/7